tests/api/testdata/db_filler.py

```diff
@@ -194,11 +202,29 @@ def _solve_field_relations(self, endpoint: Endpoints, data: dict, field: Field,
                 ][-1]
                 field.f_type.schema = build_schema_by_json(current_config_log[field.name])
 
+        elif endpoint == Endpoints.RbacNotBuiltInPolicy:
+            if field.name == "object":
+                role_fk = _data[related_field_name]
+                role = get_object_data(adcm=self.adcm, endpoint=endpoint.RbacAnyRole, object_id=role_fk)
+                _data[field.name] = field.f_type.payload = [
```

Comment:
payload is still needed?

Reply:
Yep, I use it in other places. I guess we need to cover more than one place for such "special" stuff anyway

Reply:
I've reworked `payload` a little, made some fixes, so it'll be called directly in `_prepare_field_value_for_object_creation`